modules/@apostrophecms/image/ui/apos/components/AposImageRelationshipEditor.vue

```diff
@@ -269,7 +269,8 @@ export default {
           body: {
             _id: this.item.attachment._id,
             crop: this.docFields.data
-          }
+          },
+          busy: true
```

Comment:
Unrelated fix: block the UI when saving in the manual cropping interface (crop generation takes a moment)